guiEditor/src/diagram/workbench.tsx

```diff
@@ -853,8 +854,16 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
     //Add zoom and pan controls
     addControls(scene: Scene) {
 
-        const zoomFnScrollWheel = (e: IWheelEvent) => {
-            const delta = this.zoomWheel(e);
+        const zoomFnScrollWheel = (e: IWheelEvent | WheelEvent) => {
+            let ev: any = e;

```

Comment:
Just wondering about typing - are both the interface and the implementation needed? any why is the any casting required here?

Reply:
I'm not sure if this was the best approach but since this can both be called with WheelEvents and IWheelEvents (invoking from our side), I wanted to cover our bases and basically funnel either into an IWheelEvent.  I used `any` here because our custom events now need a deviceType, deviceSlot, and inputIndex value and WheelEvents are readonly.

Reply:
reverted this